### PR TITLE
Make <img> in 1-Index-html translatable

### PR DESCRIPTION
**Pull request by ignotus666**
**Short description of changes**
This adds an option to the po4a script to make the content of the `` tag translatable - it was being skipped by po4a and made it impossible to show localised screenshots of the main mixer window on the homepage. It also makes it possible to easily add an arbitrary number of options in future if needed.

I made a small change to the EN version of 1-Index.md to trigger an update of the .po files. I think the change was necessary anyway as not everyone might have 5 users in their screenshot.

**Context: Fixes an issue? Related issues**
This problem was pointed out by @henkdegroot on Discord.

**Status of this Pull Request**
Tested and working

**What is missing until this pull request can be merged?**
Any Weblate PRs have to be merged first.

**Does this need translation?**

YES.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by jujudusud** [comment]
I will be able to modify the french translation for "serveral" instead of "five" for the 3.11 release when I will add the screenshots.

**Comment by pljones** [comment]
Do we need this for 3.11.0?

**Comment by ignotus666** [comment]
I suppose it's not essential, but merging it before the release won't do any harm either - it will make strings that aren't visible on the website anyway translatable. If people do manage to update the strings (it's just a couple, and only useful if they have localised screenshots for their language), then that's a bonus.

**Comment by pljones** [comment]
It's more that I'm thinking we shouldn't bother adding jamulussoftware/jamuluswebsite changes to the Tracking board if they're not targeting a specific release, as that's what we're Tracking on the board.  So if this is just on going maintenance, that's fine -- but I'd expect it _not_ to target "next-release"... though, I guess that's a different discussion.  Details still to be thrashed out...

File: wiki/en/misc/1-index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone
   <div class="fx-col-100-xs">
     <figure class="mainbannerfig">
       <a href="wiki/Getting-Started">
-      <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing five people from different countries connected.">
+        <img src="{% include img/en-screenshots/main-screen-medium.inc %}" style="border: 5px solid grey;" id="jamulusbanner" loading="lazy" alt="A screenshot of the main mixer window showing five people from different countries connected.">
```
Comment (softins) [inline wiki/en/misc/1-index.md]: This file needs rebasing due to the change in #1037. That should fix the conflict reported by GH below.

**Comment by ann0see** [comment]
@ignotus666 please open a new PR with your changes...

**Comment by ignotus666** [comment]
Yes, will do.

**Pull request closed without merging**